src/drivers/telemetry/frsky_telemetry/frsky_telemetry.cpp

```diff
@@ -667,30 +702,40 @@ int frsky_telemetry_main(int argc, char *argv[])
 			usleep(1000000);
 			warnx(".");
 		}
-
+		
 		warnx("terminated.");
+		device_name = NULL;
 		exit(0);
 	}
 
 	if (!strcmp(argv[1], "status")) {
 		if (thread_running) {
 			switch (frsky_state) {
+				
 			case SCANNING:
-				errx(0, "running: SCANNING");
+				PX4_INFO("running: SCANNING");
+				PX4_INFO("port: %s\n", device_name);
+				exit(0);
```

Comment:
would be better to use return 0 here as that will make this driver Linux-compatible.

Reply:
@LorenzMeier ... sorry. Used 'make format' now. Changed exit(0) with return 0. Tested with D type telemetry it works OK. Tested start -d \dev\ttyS1, status, stop then start again. Thanks @bkueng for suggestion.